src/agri_ai/line_bot/app.py

```
@@ -0,0 +1,253 @@
+        if mongo_client:
+            await mongo_client.db.command("ping")
+        
+        # Check LINE Bot API
+        if line_bot_api:
+            line_bot_api.get_bot_info()
+        
+        return {"status": "healthy"}
+    
+    except Exception as e:
+        logger.error(f"Health check failed: {e}")
+        raise HTTPException(status_code=503, detail="Service unavailable")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)